pkg/click/click

```diff
@@ -101,7 +109,7 @@ execute() {
 OPTIND=1
 
 # parse options
-while getopts ":b:ehi:jko:px" OPT; do
+while getopts ":b:ehi:jkoc:px" OPT; do
```

Comment:
This is incorrect, as it's specifying that the `-c` option must be [followed by an argument](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#index-getopts). In addition, the options are usually alphabetically ordered for `getopts` as well.
```
:b:cehi:jko:px
```

Reply:
changed to be what you posted